akvo/rsr/spa/app/modules/editor/section5/results/validations.js

```diff
@@ -0,0 +1,19 @@
+import * as yup from 'yup'
+
+const RSR = yup.object().shape({
```

Comment:
This is the validations we currently have defined in the DB.  RSR doesn't have any validations, but it makes sense to have the same validations as IATI, except probably not "enforcing the minimum one indicator per result" rule. 


# IATI<a id="sec-2"></a>

## Mandatory<a id="sec-2-1"></a>

- [ ] At least one indicator per result
- [ ] `rsr_indicator.title`
- [ ] `rsr_indicator.measure`
- [ ] `rsr_indicatorperiod.period_end`
- [ ] `rsr_indicatorperiod.period_start`
- [ ] `rsr_result.title`
- [ ] `rsr_result.type`

# DGIS<a id="sec-3"></a>

## Mandatory<a id="sec-3-1"></a>

- [ ] At least one indicator
- [ ] `rsr_indicator.baseline_value`
- [ ] `rsr_indicator.baseline_year`
- [ ] `rsr_indicator.measure`
- [ ] `rsr_indicator.title`
- [ ] At least one indicatorperiod
- [ ] `rsr_indicatorperiod.period_end`
- [ ] `rsr_indicatorperiod.period_start`
- [ ] `rsr_indicatorperiod.target_value`
- [ ] At least one result
- [ ] `rsr_result.title`
- [ ] `rsr_result.type`